README.md

```diff
@@ -41,8 +41,8 @@ Check out the [user guide](USAGE.md) for more information on what can be done.
 
 ## Development
 
-Opcard uses [virtualsmartcard](https://frankmorgner.github.io/vsmartcard/) for testing.
-`make test` will run `opcard` on the host through virtualsmartcard and test it.
+Opcard uses [vpiccsmartcard](https://frankmorgner.github.io/vsmartcard/) for testing.
+`make test` will run `opcard` on the host through vpiccsmartcard and test it.
```

Comment:
I’d either use `vsmartcard` (= the entire project) or `vpicc` (= the specific Rust crate) here (or maybe both)?

Reply:
I used search and replace and then checked which replacements were incorrect and missed this.